compiler/src/dotty/tools/dotc/parsing/Parsers.scala

```diff
@@ -3203,13 +3200,6 @@ object Parsers {
                 if !idOK then syntaxError(i"named imports cannot follow wildcard imports")
                 namedSelector(termIdent())
         }
-        val rest =
```

Comment:
It seems that changes the accepted set of programs since it does not propagate `isWildcard` to following import selectors.